client/blocks/eligibility-warnings/index.jsx

```diff
@@ -161,9 +185,14 @@ EligibilityWarnings.defaultProps = {
 
 const mapStateToProps = state => {
 	const siteId = getSelectedSiteId( state );
+	const eligibilityData = getEligibility( state, siteId );
+	const dataLoaded = !! eligibilityData.lastUpdate;
```

Comment:
Good point! I was considering time based check, but was not sure what value to use (or if we have some defaults for similar cases in Calypso), and also if that check should be hard-coded here or added in schema validation as you've mentioned. 

In any case, I think that `X minutes` shouldn't be too long because I suspect that the most frequent use-case will be a user that has just fixed a blocker, and is reloading this check to `Proceed` with AT.